test/good.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>My favourite title!</title>
+  </head>
+  <body>
+    <h1 class="bigtitle">My other favourite title A ...</h1>
+    <h1 class="bigtitle">My other favourite title B ...</h1>
+    <h1 class="smalltitle">My other favourite title C ...</h1>
+    <p id="main-paragraph">What a wonderful webpage!</p>
+    <input type="checkbox" checked>
+    <input type="checkbox" checked="checked">
+    <input type="checkbox">
+  </body>
+</html>
```

Comment:
I'd like to use a more "realistic" document. Having three `<h1>`s is a bit silly. ;)

For testing `children` it would be useful to have a list of some kind. For example:

```html
<ol>
  <li>one</li>
  <li>two</li>
  <li>three</li>
  <li>four</li>
  <li>five</li>
</ol>
```